api/internal/builtins/PrefixTransformer.go

```diff
@@ -1,37 +1,34 @@
-// Code generated by pluginator on PrefixSuffixTransformer; DO NOT EDIT.
+// Code generated by pluginator on PrefixTransformer; DO NOT EDIT.
 // pluginator {unknown  1970-01-01T00:00:00Z  }
 
 package builtins
 
 import (
 	"errors"
 
-	"sigs.k8s.io/kustomize/api/filters/prefixsuffix"
+	"sigs.k8s.io/kustomize/api/filters/prefix"
+
 	"sigs.k8s.io/kustomize/api/resmap"
 	"sigs.k8s.io/kustomize/api/types"
 	"sigs.k8s.io/kustomize/kyaml/resid"
-	"sigs.k8s.io/yaml"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
 )
 
-// Add the given prefix and suffix to the field.
-type PrefixSuffixTransformerPlugin struct {
+// Add the given prefix to the field
+type PrefixTransformerPlugin struct {
 	Prefix     string        `json:"prefix,omitempty" yaml:"prefix,omitempty"`
-	Suffix     string        `json:"suffix,omitempty" yaml:"suffix,omitempty"`
 	FieldSpecs types.FsSlice `json:"fieldSpecs,omitempty" yaml:"fieldSpecs,omitempty"`
 }
 
-// A Gvk skip list for prefix/suffix modification.
-// hard coded for now - eventually should be part of config.
```

Comment:
Let's keep the comment here

```
// A Gvk skip list for prefix modification.
// hard coded for now - eventually should be part of config.
```

Reply:
Make sure we also have this comment in `SuffixTransformer.go`. Also I realized I commented on the generated file, so please make the change in the corresponding files in the `plugins` directory.